jupyterhub_singleuser_profiles/profiles.py

```diff
@@ -99,13 +99,21 @@ def update_user_profile_cm(self, username, data={}):
     user_cm = self.get_user_profile_cm(username)
     cm_name = _USER_CONFIG_MAP_TEMPLATE % escape(username)
     cm_key_name = "profile"
-    data = json.loads(data)
     for key, value in data.items():
       user_cm[key] = value
     self.write_config_map(cm_name, cm_key_name, user_cm)
 
   def get_user_profile_cm(self, username):
-    return self.read_config_map(_USER_CONFIG_MAP_TEMPLATE % escape(username), "profile")
+    cm = self.read_config_map(_USER_CONFIG_MAP_TEMPLATE % escape(username), "profile")
+    if cm == {}:
+      return {
+        "env":{"AWS_ACCESS_KEY_ID":"", "AWS_SECRET_ACCESS_KEY":""},
```

Comment:
Can we put this somwhere at the top of the file as `_DEFAULT_USER_CM` so that it is not burried in the code?